<ui version="4.0" >
 <class>Call</class>
 <widget class="QDialog" name="Call" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>354</width>
    <height>180</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="lb_to" >
       <property name="text" >
        <string>To:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le_to" />
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="ck_useVideo" >
     <property name="text" >
      <string>Use video</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="lb_bandwidth" >
       <property name="text" >
        <string>Sending bandwidth:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cb_bandwidth" />
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="lb_status" >
     <property name="text" >
      <string>Call status</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>336</width>
       <height>16</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="Line" name="line" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="BusyWidget" native="1" name="busy" />
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>111</width>
         <height>28</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pb_reject" >
       <property name="text" >
        <string>&amp;Reject</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pb_accept" >
       <property name="text" >
        <string>&amp;Accept</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BusyWidget</class>
   <extends>QWidget</extends>
   <header>busywidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>le_to</tabstop>
  <tabstop>ck_useVideo</tabstop>
  <tabstop>cb_bandwidth</tabstop>
  <tabstop>pb_reject</tabstop>
  <tabstop>pb_accept</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
